src/ContentMessages.js

```diff
@@ -25,15 +25,19 @@ import sdk from './index';
 import { _t } from './languageHandler';
 import Modal from './Modal';
 import RoomViewStore from './stores/RoomViewStore';
-
 import encrypt from "browser-encrypt-attachment";
+// eslint-disable-next-line camelcase
+import png_chunks_extract from "png-chunks-extract";
```

Comment:
`import * as extractPngChunks` or something to shut up eslint?

Reply:
Doesn't seem to work, as `png-chunks-extract` uses commonjs. With:
```
import * as extractPngChunks from "png-chunks-extract";
```
I get an object with the property `default` having the function I want. Not sure how to write this as an import expression.

Reply:
When you are importing the default export of a module, you get to choose the name you'll refer to it as. It doesn't need to match the name in the exporting module. So, you could use:

```suggestion
import extractPngChunks from "png-chunks-extract";
```

to avoid the lint issue. This is _without_ `* as` since you only need the default export. See [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) for more gory details.